graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -104,7 +104,7 @@ public static class GraphiQLConfiguration implements Toggleable {
         /**
          * The configuration name whether GraphiQL is enabled.
          */
-        public static final String ENABLED = GraphQLConfiguration.PREFIX + "." + PREFIX + ".enabled";
+        public static final String ENABLED_CONFIG = GraphQLConfiguration.PREFIX + "." + PREFIX + ".enabled";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.